envoy/api/v2/lds.proto

```diff
@@ -132,4 +132,8 @@ message Listener {
   // case the original source and destination addresses and ports are preserved on accepted
   // connections. Requires Envoy to run with the *CAP_NET_ADMIN* capability. Defaults to false.
   bool transparent = 10;
+
+  // Whether the listener should accept TCP Fast Open (TFO) connections.
+  // Supports Linux and OSX. Defaults to false.
```

Comment:
I'd drop the OS whitelist and only add a blacklist if needed.

Reply:
Will do.